### DISCUSSION TITLE
Cannot find InvenTree image on DigitalOcean

### DISCUSSION THREAD

**Discussion by owenwnew** [inventree/InvenTree#7432] [General]
The link in the page https://inventree.org/digitalocean pointing to the digital ocean market place is not working. It returns a page not found error from Digital Ocean. 
I also tried to search InvenTree in the digital ocean market place and result nothing. Am I missing something here?

**Comment by matmair** [comment]
I am waiting for the image to be re-approved after I pushed a large update

**Reply by owenwnew** [reply]
Thank you for letting me know!

**Reply by TooperProIT** [reply]
> I am waiting for the image to be re-approved after I pushed a large update

Do you know usually how long will it take get it approved? My project is kinda waiting on this image to be avaliable.

**Reply by matmair** [reply]
No, and this is low on my priority list.
